docs/new_packages.md

```diff
@@ -148,3 +148,51 @@ Shell commands to run after building the library. These are run inside of
 A list of required packages.
 
 (Unlike conda, this only supports package names, not versions).
+
+## Manual creation of a Pyodide package
```

Comment:
We need to change this title to reflect that this is an advanced setup and not something most users will want to do. And also mention that for pure python packages creating Python wheels and loading them with micropip is preferable.